R/agglomerate.R

```diff
@@ -45,26 +45,26 @@
 #'   to \code{getPrevalence} and \code{getPrevalentTaxa} and used in
 #'   \code{agglomeratebyPrevalence}
 #'   \itemize{
-#'        \item \code{remove_empty_ranks}: A single boolean value for selecting
+#'        \item \code{empty.ranks.rm}: A single boolean value for selecting
 #'        whether to remove those columns of rowData that include only NAs after
-#'        agglomeration. (By default: \code{remove_empty_ranks = FALSE})
-#'        \item \code{make_unique}: A single boolean value for selecting
-#'        whether to make rownames unique. (By default: \code{make_unique = TRUE})
+#'        agglomeration. (By default: \code{empty.ranks.rm = FALSE})
+#'        \item \code{make.unique}: A single boolean value for selecting
+#'        whether to make rownames unique. (By default: \code{make.unique = TRUE})
 #'        \item \code{detection}: Detection threshold for absence/presence.
 #'        Either an absolute value compared directly to the values of \code{x}
-#'        or a relative value between 0 and 1, if \code{as_relative = FALSE}.
+#'        or a relative value between 0 and 1, if \code{as.relative = FALSE}.
 #'        \item \code{prevalence}: Prevalence threshold (in 0 to 1). The
 #'        required prevalence is strictly greater by default. To include the
-#'        limit, set \code{include_lowest} to \code{TRUE}.
+#'        limit, set \code{include.lowest} to \code{TRUE}.
 #'        \item \code{as.relative}: Logical scalar: Should the detection
 #'        threshold be applied on compositional (relative) abundances?
 #'        (default: \code{FALSE})
-#'        \item \code{mergeRefSeq} \code{TRUE} or \code{FALSE}: Should a
+#'        \item \code{agglomerate.refseq} \code{TRUE} or \code{FALSE}: Should a
```

Comment:
agglomerate matches the other things happening at the same time but the process  is not really "agglomeration" here. Should we have something like `update.refseq` or `join.refseq` here?  or is it too confusing

Reply:
Hmm, yes. Maybe:

`update.tree` and `update.refseq`

Reply:
@Daenarys8

Reply:
does this apply to all agglomerate.tree occurrences or just for R/agglomerate.R?

Reply:
All